hw/ip/spi_device/doc/_index.md

```diff
@@ -866,8 +866,12 @@ The SW may configure TPM_CFG.hw_reg_dis and/or TPM_CFG.invalid_locality to fully
 
 ### TPM mode: FIFO and CRB
 
-The HW returns the return-by-HW registers for the read request in the TPM FIFO mode.
-In the TPM CRB mode (TPM_CFG.tpm_mode is 1), the logic always upload the command and address to the SW and waits for the read FIFO data even the received address falls into the managed address.
+The TPM protocol supports two protocol interfaces: FIFO and CRB (Command Response Buffer).
```

Comment:
@eunchan can you have a look here? I reworded this a bit since i was getting really confused at first.